Documentation/DebugJavascriptDevTools.md

```diff
@@ -0,0 +1,29 @@
+# BabylonNative Script Debugging on Win32
+You can use Chromium DevTools to debug your JavaScript while running BabylonNative on Win32 (x86/x64) with the V8 javascript engine.
+
+## Set up BabylonNative to Use V8
+When running CMake, be sure to specify the following flag:
+`-D NAPI_JAVASCRIPT_ENGINE=V8`
+
+For example:
+```powershell
+mkdir Build
+cd Build
+cmake -A Win32 -D NAPI_JAVASCRIPT_ENGINE=V8 ..
+```
+
+## Add the Remote Target URL
+Depending on which Chromium-based browser you're using (Edge or Chrome), navigate to `edge://inspect` or `chrome://inspect`.
+Click `Configure...` and in the target discovery settings add an entry `localhost:{port}`, where `{port}` is consistent with the one supplied to the call to [`StartInspector`](https://github.com/BabylonJS/BabylonNative/blob/2db465f5569d8eb833245a41030c5219cfa3fe59/Apps/Playground/Win32/App.cpp#L141-L144).
```

Comment:
>StartInspector [](http://example.com/codeflow?start=159&length=14)

nit: this is a cool hyper link but we should say what start inspector does in this comment